bazel/target_recipes.bzl

```diff
@@ -2,7 +2,6 @@
 # target in //ci/prebuilt/BUILD to the underlying build recipe in
 # ci/build_container/build_recipes.
 TARGET_RECIPES = {
-    "ares": "cares",
```

Comment:
Should we also remove https://github.com/envoyproxy/envoy/blob/master/ci/build_container/build_recipes/cares.sh?